src/mmrelay/matrix_utils.py

```
@@ -1724,6 +1636,142 @@ async def _resolve_alias(alias: str) -> str | None:
+                        "Invite handling is disabled for subsequent syncs."
+                    )
+                except Exception:
+                    logger.exception("Invite-safe sync retry failed")
+        except ImportError:
+            logger.debug("Invite-safe sync retry handler failed", exc_info=True)
+        if sync_response is None:
+            logger.exception("Matrix sync failed")
+            try:
+                if matrix_client:
+                    await matrix_client.close()
+            except Exception:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)